### DISCUSSION TITLE
Back button after searching *feature request*

### DISCUSSION THREAD

**Discussion by sodepop** [Kunzisoft/KeePassDX#1180] [Ideas]
First of, great app! This is my main KeePass app for Android. Works perfectly. However, would it be possible to add a function where when you do a search in a database, hit enter, go into an entry, and then when you go back (hit system back key or top left back arrow in app)...the search results are still there? Currently, when you hit back, it goes to the database default view and your search results are wiped out.

Just a little feature that would be helpful. Thanks!

**Comment by J-Jamet** [comment]
Done in version 3.3.0
